pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopics.java

```diff
@@ -632,19 +635,22 @@ void run() throws Exception {
                 "--force" }, description = "Close all producer/consumer/replicator and delete topic forcefully")
         private boolean force = false;
 
-        @Parameter(names = { "-d",
-                "--deleteSchema" }, description = "Delete schema while deleting topic")
+        @Parameter(names = {"-d", "--deleteSchema"}, description = "Delete schema while deleting topic, "
+                + "but the parameter is invalid and the schema is always deleted", hidden = true)
         private boolean deleteSchema = false;
```

Comment:
Should also update the default value to true?

Reply:
Maybe we can't change the default value to `true` because that would change its usage.
Before modification: it means false if you don't specify `-d`, and it means true if you specify `-d`;
After modification: it means true if you don’t specify `-d`, but it means false only you specify `-d false`.
The two are not fully compatible.
It is similar to #12128